ut/alarm_tables_testing.hpp

```diff
@@ -6,6 +6,8 @@
 class CustomDefs: public SNMPTest
 {
 public:
+  CustomDefs() {alarm_address = "16162";}
```

Comment:
Should you use an [initializer list](http://en.cppreference.com/w/cpp/language/initializer_list) here? We generally prefer that when a constructor is just initializing variables, as they're specifically for that.